docrepr/sphinxify.py

```diff
@@ -203,10 +203,11 @@ def generate_extensions(render_math):
     """Generate list of Sphinx extensions"""
     # We need jsmath to get pretty plain-text latex in docstrings
     extensions = []
-    if sphinx.__version__ < "1.1" or not render_math:
-        extensions = ['sphinx.ext.jsmath']
-    else:
-        extensions = ['sphinx.ext.mathjax']
+    if render_math:
+        if sphinx.__version__ < "1.1":
+            extensions = ['sphinx.ext.jsmath']
+        else:
+            extensions = ['sphinx.ext.mathjax']
```

Comment:
```suggestion
        extensions = ['sphinx.ext.mathjax']
```

Sphinx 1.1 was released over a decade ago, and most likely `docrepr` doesn't even work with Sphinx that old anymore anyway, so we may as well just simplify this...

Reply:
Yep :) There are mentions to jsmath in the config that we should maybe remove as well?